src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -1,42 +1,38 @@
-import React, { Component } from 'react';
-import { withRouter } from 'react-router-dom';
-import { Gallery, GalleryItem } from '@patternfly/react-core';
-import { PageHeader, PageHeaderTitle, Main } from '@red-hat-insights/insights-frontend-components';
-
-import asyncComponent from '../../Utilities/skeletonAsyncCard';
+/* eslint-disable no-console */
 import './_dashboard.scss';
 
-const ComplianceCard = asyncComponent(() => import ('../../SmartComponents/Cards/ComplianceCard'));
-const VulnerabilityCard = asyncComponent(() => import ('../../SmartComponents/Cards/VulnerabilityCard'));
+import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
+import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
+import { TemplateCard, TemplateCardBody, TemplateCardHeader } from '../Template/TemplateCard';
+
+import { Main } from '@red-hat-insights/insights-frontend-components/components/Main';
+import React from 'react';
+import asyncComponent from '../../Utilities/skeletonAsyncCard';
 
-/**
- * A smart component that handles all the api calls and data needed by the dumb components.
- * Smart components are usually classes.
- *
- * https://reactjs.org/docs/components-and-props.html
- * https://medium.com/@thejasonfile/dumb-components-and-smart-components-e7b33a698d43
- */
-class Dashboard extends Component {
+const ComplianceCard = asyncComponent(() => import('../../SmartComponents/Compliance/ComplianceCard'));
+const VulnerabilityCard = asyncComponent(() => import('../../SmartComponents/Vulnerability/VulnerabilityCard'));
 
-    render() {
-        return (
-            <React.Fragment>
-                <PageHeader>
-                    <PageHeaderTitle title='Health of your infrastructure'/>
-                </PageHeader>
-                <Main className='ins-l-dashboard'>
-                    <Gallery gutter='md'>
-                        <GalleryItem>
-                            <VulnerabilityCard />
-                        </GalleryItem>
-                        <GalleryItem>
-                            <ComplianceCard />
-                        </GalleryItem>
-                    </Gallery>
-                </Main>
-            </React.Fragment>
-        );
-    }
-}
+const Dashboard = () =>
+    <React.Fragment>
+        <PageHeader>
+            <PageHeaderTitle title='Health of your infrastructure' />
+        </PageHeader>
+        <Main className='ins-l-dashboard'>
+            <Gallery gutter='md'>
+                <GalleryItem>
+                    <VulnerabilityCard />
+                </GalleryItem>
+                <GalleryItem>
+                    <ComplianceCard />
+                </GalleryItem>
+                <GalleryItem>
```

Comment:
I don't think that this card should be here.

Reply:
it'll be changed up for insights one in a lil bit, but if yah don't want it in the interim i can remove

Reply:
Yes please, so we won't propagate test things to end users.

Reply:
wait, this is live? where is this live!!

Reply:
(also fixed)

Reply:
cloud.redhat.com/beta/rhel/dashboard

Reply:
OH wow, ok, I thought this was... not in use 😬  n was gonna be the thing we are gonna use to replace insights dashboard 🤔 gonna have to plan the rollout of new cards a lil strategically

Reply:
@ryelo so whats the plan for work on this given its live?

Reply:
I've changed the custom_release so that we don't push this to prod-beta. We can make changes in ci/qa-beta and it should be fine.

Reply:
right on! 

OK GONNA KEEP ON TEARING IT UP THEN... just in another pr, done with this one, please end it 

![external-content duckduckgo com](https://user-images.githubusercontent.com/6640236/74843374-cc87af00-52f9-11ea-9c08-7416d36d3ad0.jpg)